lib/routes.json

```diff
@@ -8783,26 +9042,12 @@
         "accept": "application/vnd.github.cryptographer-preview"
       },
       "method": "POST",
-      "params": {
-        "armored_public_key": {
-          "required": true,
-          "type": "string"
-        }
-      },
+      "params": {},
```

Comment:
this has been acknowledged by the GitHub docs team and should get fixed soon. Not worth to work around it